types/react-native-awesome-card-io/index.d.ts

```diff
@@ -0,0 +1,74 @@
+// Type definitions for react-native-awesome-card-io 0.8
+// Project: https://github.com/Kerumen/react-native-awesome-card-io
+// Definitions by: Onur Var <https://github.com/OnurVar>
+//                 Emlyn Bolton <https://github.com/emlynmac>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.6
```

Comment:
This should be the minimum version of TypeScript that is compatible with these type definitions - this seems like it's too high.

Reply:
Hi @uniqueiniquity, I've updated my PR with the requested changes.